app/src/main/java/app/foodfinderapp/dto/Menu.kt

```diff
@@ -1,6 +1,8 @@
 package app.foodfinderapp.dto
 
-data class Menu(var menuID : Int, var menuFoods : ArrayList<Food>, var menuCategories : String) {
+import com.google.gson.annotations.SerializedName
+
+data class Menu(var menuID : Int, var menuFoods : ArrayList<Food>,@SerializedName("manuCategories") var menuCategories : String) {
```

Comment:
See notes above.

@SerializedName is only required if the JSON field names do not match the data class attribute names.
In a case like this, where the names match, @SerializedName is not providing value... and thus, is extra code that is not needed.